convert2rhel/toolopts.py

```diff
@@ -375,6 +375,9 @@ def _process_cli_options(self):
         if (config_opts.activation_key or config_opts.password) and parsed_opts.password_from_file:
             loggerinst.warning("Password file take precedence over the config file.")
 
+        if (tool_opts.org and not tool_opts.activation_key) or (not tool_opts.org and tool_opts.activation_key):
+            loggerinst.critical("You haven't passed organization with activation key. Please pass both.")
```

Comment:
So, I did a bit of rephrasing in the message to sound more “generic” and not assume that `organization` is present and `acitivation key` is not (or the other way around).

```suggestion
            loggerinst.critical("Either organization or activation key are missing. You have to use both of them.")
```

If you feel that it doesn't make sense, feel free to resolve this comment.

Reply:
Changed!